Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG BUILD_NUMBER
 ENV BUILD_NUMBER=${BUILD_NUMBER:-1_0_0}
 
 WORKDIR /builder
-COPY csr-api-${BUILD_NUMBER}.jar app.jar
+COPY csr-api-*.jar app.jar
```

Comment:
I'd assumed that the `BUILD_NUMBER` would be the same, however the gradle build sets it to the current date (e.g. `csr-api-2025-07-29.jar`) whereas by the time we reach the Dockerfile it is `csr-api-2025-07-29.31.683957c.jar`.
I *could* do something with the date, but the build could span midnight and then we'd be stuck.  I've therefore gone back to the wildcard since there should only be one file anyway.